src/class/hash.rs

```diff
@@ -6,7 +6,7 @@ use types::{Value, ValueType};
 use {AnyObject, Object, VerifiedObject};
 
 /// `Hash`
-#[derive(Debug, PartialEq)]
+#[derive(Copy, Debug, PartialEq)]
```

Comment:
Introducing `Copy` may lead to some unexpected behaviour for compound objects like `Hash`. For example, if a hash is copied, the compiler will not give any warnings, and a user may spend some hours debugging the program and trying to find out why the hash does not look as it is expected. I'd like to stick to default Rust behaviour and use the full power of borrow checker :)

Also I am going to change some parts of API which take objects by values instead of by references and implement the `Copy` trait only for the primitive types (`Fixnum`, `Float`, `Symbol` etc, which always have the same `object_id` for the same object) similarly to Rust.